tests/build/chains.go

```diff
@@ -130,7 +131,8 @@ var _ = framework.ChainsSuiteDescribe("Tekton Chains E2E tests", Label("ec", "HA
 			GinkgoWriter.Printf("The image signed by Tekton Chains is %s\n", imageWithDigest)
 		})
 
-		It("creates signature and attestation", func() {
+		// Skip until https://issues.redhat.com/browse/RHTAPBUGS-276 is fixed
+		It("creates signature and attestation", Pending, func() {
```

Comment:
we'll need a different mechanism for checking image attestation and signature (since we're no longer using openshift-registry for storing the built image).